IRescue/Core/DataTypes/RotationMatrix.cs

```diff
@@ -90,5 +95,34 @@ private static float[] CreateMatrixVector(float xRotation, float yRotation, floa
                 };
             return rot;
         }
+
```

Comment:
These two private methods have no calling hierachy, is there a specific reason for them to be here for feature use or should they be removed?